README.md

```diff
@@ -1299,8 +1299,8 @@ Work in progress drivers. Help the authors make these crates awesome!
 1. [adafruit-bluefruit-protocol]: A `no_std` parser for the [Adafruit Bluefruit LE Connect controller protocol]. - [![crates.io](https://img.shields.io/crates/v/adafruit-bluefruit-protocol)](https://crates.io/crates/adafruit-bluefruit-protocol)
 1. [wtx](https://github.com/c410-f3r/wtx): Among other things, provides implementations to interact with HTTP, WebSockets and Databases.
 
+[no-std-category]: https://crates.io/categories/no-std
```

Comment:
Why this?

Reply:
It is cleaning up the ordering of the sections. It was in the wrong place.

Reply:
Ok. I guess I was (am?) confused about how the formatting works — apologies.